tutorindigo/templates/indigo/lms/templates/index.html

```diff
@@ -47,10 +49,68 @@
         </div>
 
       </header>
-      <%include file="${courses_list}" />
 
     </section>
 </main>
+<div class="home-container">
+  <h2 class="new-title">Universidades</h2>
+  <div class="universities-grid">
+      <div class="university">
+          <a href="${reverse('courses')}?filtertype=org&filterquery=UAH" rel="noopener">
+              <img src="${static.url('images/logo_uah.png')}" alt="Universidad de Alcalá">
```

Comment:
Are these images part of the plugin or will be upload in a different way?

Reply:
They have the images as part of their theme folder (in this case, inside the theme folder within the tutor-plugin). I added them there to follow their practice, but if you think of a better way, please let me know.

Reply:
I forgot to add the images. I already added them.